tests/math_expression_extraction/test_donut_extractor.py

```
@@ -23,9 +23,9 @@ def main():
     image_path = "tests/math_expression_extraction/assets/math_equation.png"
 
     result = extractor.extract(image_path)
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)